pkgs/development/tools/yarn2nix-moretea/yarn2nix/lib/fixPkgAddMissingSha512.js

```diff
@@ -46,21 +46,21 @@ async function fixPkgAddMissingSha1(pkg) {
     return pkg
   }
 
-  const [url, sha1] = pkg.resolved.split('#', 2)
+  const [url, sha512] = pkg.resolved.split('#', 2)
```

Comment:
As mentioned, I am not sure if `resolved` field can contain anything other than `sha1`. Will prolly need to scour https://github.com/yarnpkg/yarn